src/config/print/config.md

```diff
@@ -30,15 +17,31 @@ configure a printer using the command line.
 
 ## Graphical interface
 
-To configure the printer using a native GUI interface, run the following:
+The `system-config-printer` package offers simple and robust configuration of
+new printers. Install the package:
 
 ```
 # xbps-install -S system-config-printer
-# system-config-printer
 ```
 
-This command does not need to be run as root if PolicyKit is in use and
-`cups-pk-helper` is installed.
+Normally this tool would require root privileges to configure printers, but
+unprivileged access through `polkit` can be achieved with a helper. If you are
+interested in this, install the helper:
+
+```
+# xbps-install -S cups-pk-helper
+```
+
+You may run `system-config-printer` from your desktop environment's application
+launcher, or via the command line:
+
+```
+$ system-config-printer
+```
+
+If you are not using the `polkit` helper mentioned above, the command must be
+invoked as root. Various packages can be used invoke GUI applications as root,
+including `lxqt-sudo` and `kdesu`.
```

Comment:
I think this is slightly outside of the scope of this page. This handbook has so far been assuming basic proficiency of being able to launch both user and superuser commands from a terminal, and we might be better off sticking to that assumption here.